src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -107,8 +107,7 @@ public void run() {
 				DrawerFeature.setupDrawerPlugin(app);
 				CommandsFeature.setupCommandManager();
 
-				DriverFeature.setupDriverPlugin(app);
-				DriverFeature.addDriverChangeObserver(new DriverChangeObserver(app));
+				DriverFeature.setupDriverPlugin(app, new DriverNameUpdateObserver(app));
```

Comment:
new DriverNameUpdateObserver(app) please put it also in setup method! (as a separate variable!)